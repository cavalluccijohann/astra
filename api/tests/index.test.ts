import { describe, expect, it } from 'vitest'
import { $fetch } from 'nitro-test-utils/e2e'

describe('API is alive', () => {
  it('should return a 200 status', async () => {
    const { data, status } = await $fetch('/')

    expect(status).toBe(200)
    expect(data).toMatchSnapshot()
  })
})

describe('Feed route', () => {
  it('should return a 200 status', async () => {
    const response = await $fetch('/album', {
      method: 'GET',
    })

    expect(response.status).toBe(200)
  })
})
