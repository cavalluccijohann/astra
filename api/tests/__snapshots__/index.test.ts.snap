// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`API is alive > should return a 200 status 1`] = `"Hello, world!"`;
